packages/next-bundle-analyzer/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (pluginOptions = { enabled: true }) => (nextConfig = {}) => {
+  return Object.assign({}, nextConfig, {
+    webpack (config, options) {
+      if (pluginOptions.enabled) {
```

Comment:
Since we destructure this up top we can just do this now:

```suggestion
      if (enabled) {
```